### PR TITLE
AMDGPU: Add baseline test for vectorize of integer min/max

### PR DESCRIPTION
**Pull request by arsenm**

### CODE REVIEW

**Comment by arsenm** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/llvm/llvm-project/100513?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#100514** <a href="https://app.graphite.dev/github/pr/llvm/llvm-project/100514?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#100513** <a href="https://app.graphite.dev/github/pr/llvm/llvm-project/100513?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#97463** <a href="https://app.graphite.dev/github/pr/llvm/llvm-project/97463?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @arsenm and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: llvm/test/Transforms/SLPVectorizer/AMDGPU/min_max.ll [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+declare i16 @llvm.umax.i16(i16, i16) #0
+declare i16 @llvm.smin.i16(i16, i16) #0
+declare i16 @llvm.smax.i16(i16, i16) #0
+
+declare i32 @llvm.umin.i32(i32, i32) #0
+declare i32 @llvm.umax.i32(i32, i32) #0
+declare i32 @llvm.smin.i32(i32, i32) #0
+declare i32 @llvm.smax.i32(i32, i32) #0
+
+attributes #0 = { nounwind readnone speculatable willreturn }
```
Comment (jayfoad) [inline llvm/test/Transforms/SLPVectorizer/AMDGPU/min_max.ll:366]: Don't need any of this.

**Review by jayfoad — APPROVED** [review]
LGTM.

**Comment by arsenm** [comment]
### Merge activity

* **Jul 25, 4:25 PM EDT**: @arsenm started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/llvm/llvm-project/100513).

**Pull request merged**